#!/bin/sh -e
##  by Sebastien Delafond <sdelafond@gmx.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: zonecheck-webpages.dpatch
## DP: - Makes direct link from form.html.en to form.html.fr and vice-versa

Index: zonecheck-3.0.0/www/html/form.html.en
===================================================================
--- zonecheck-3.0.0.orig/www/html/form.html.en	2010-06-24 11:09:51.000000000 +0200
+++ zonecheck-3.0.0/www/html/form.html.en	2010-06-24 11:09:57.000000000 +0200
@@ -70,7 +70,7 @@
   <body>
 
 
-    <div class="paragraph" id="lang">Lang: <b>EN</b> / <a href="/zonecheck/fr/">FR</a></div>
+    <div class="paragraph" id="lang">Lang: <b>EN</b> / <a href="/zonecheck/fr/form.html">FR</a></div>
 
     <!-- Logo -->
     <h1 id="logo">
Index: zonecheck-3.0.0/www/html/form.html.fr
===================================================================
--- zonecheck-3.0.0.orig/www/html/form.html.fr	2010-06-24 11:09:27.000000000 +0200
+++ zonecheck-3.0.0/www/html/form.html.fr	2010-06-24 11:09:39.000000000 +0200
@@ -68,7 +68,7 @@
   <body>
 
 
-    <div class="paragraph" id="lang">Lang: <a href="HTML_PATH/en/">EN</a> / <b>FR</b></div>
+    <div class="paragraph" id="lang">Lang: <a href="HTML_PATH/en/form.html">EN</a> / <b>FR</b></div>
 
     <!-- Logo -->
     <h1 id="logo">
@@ -571,4 +571,4 @@
 <!-- mode: html       -->
 <!-- End:             -->
 
-</html>
\ No newline at end of file
+</html>
